creditcoin-js/src/testUtils/index.ts

```diff
@@ -26,6 +26,8 @@ export type TestData = {
 export const testData = (ethereumChain: Blockchain, createWalletF: CreateWalletFunc): TestData => {
     return {
         blockchain: ethereumChain,
+        // WARNING: don't hard-code this, see examples/loan-cycle.ts
+        // NOTICE: this function is only used during testing before the Creditcoin API has been initialized
         expirationBlock: 10_000_000,
```

Comment:
Let me know if you'd like me to refactor this? 

The test suite as-is will start failing transaction submissions past block 10 mil (when running against testnet/mainnet). Probably not going to be a practical problem for a long time.

Reply:
I think the warning here is probably enough. LGTM